### ISSUE TITLE
Implement Combine support for Firebase

### ISSUE THREAD

**Issue by peterfriese** [firebase/firebase-ios-sdk#7295]
### SDKs to support
- [ ] Authentication
- [ ] Realtime Database
- [ ] Cloud Firestore
- [ ] Cloud Storage
- [ ] Cloud Functions
- [ ] Machine Learning
- [ ] Cloud Messaging
- [ ] Remote Config
- [ ] Dynamic Links

**Comment by ziadtamim** [comment]
@peterfriese I will support Cloud Storage.

**Comment by peterfriese** [comment]
> @peterfriese I will support Cloud Storage.

Great - thanks, Ziad. I penciled you in - once you send a draft PR, I will add the PR to our project tracker.

**Comment by paulb777** [comment]
@peterfriese Should we close and open separate issue for specific items?

**Comment by peterfriese** [comment]
Yes, we should to that. I'll spend some time next week to clean up the tracker and set up new issues.

**Comment by mortenbekditlevsen** [comment]
I still have a PR for initial Combine support in FirebaseDatabase. Let me know if that's of any use:

https://github.com/firebase/firebase-ios-sdk/pull/7544

**Comment by ziadtamim** [comment]
There is a PR for `Decodable QuerySnapshot publisher` https://github.com/firebase/firebase-ios-sdk/pull/7549

**Comment by ncooke3** [comment]
Closing this as the focus has shifted from Combine to Swift Concurrency.
